website/src/translation/en-US.tsx

```diff
@@ -0,0 +1,198 @@
+import type { TooltipProps } from "../components/ui/Tooltip";
+import type { TranslationComponent, TranslationObject } from "./";
+
+export const translation: TranslationObject = {
+	htmlLang: () => "en",
+	documentationTitle: () => "Typst Documentation (English Version)",
+	close: () => "Close",
+	closeMenu: () => "Close menu",
+	closeSearch: () => "Close search",
+	openMenu: () => "Open menu",
+	openSearch: () => "Open search",
+	showInformation: (props: { name: string }) =>
+		`Show details for ${props.name}`,
+	tooltipKind: (props: { kind: TooltipProps["kind"] }) => {
+		switch (props.kind) {
+			case "element":
+				return "Element function";
```

Comment:
```suggestion
				return "Element";
```

These _function_ and _parameter_ suffixes could be stripped. Unlike 関数 and 引数 in Japanese, English words are much longer. In addition, the context is quite clear and these suffixes are redundant.

<img width="300" alt="Image" src="https://github.com/user-attachments/assets/f0907930-019b-413b-9d24-d0b1960dab1f" />

<img width="300" alt="Image" src="https://github.com/user-attachments/assets/3ef762b3-4ac8-4e96-afa0-a1ebf60be1b2" />